tests/tests/clobber-repair.sh

```diff
@@ -14,6 +14,7 @@ CLOBBERS=(
 "PB_SB_HDR_CRC_INVALID,"
 "PB_SB_BAD_FLAG,"
 "PB_QCONF_WRONG_VERSION"
+"PB_QSLOT_BAD_ADDR -D family",
```

Comment:
This is incorrectly formatted. Maybe the description above this list isn't verbose enough, but you want this line to look exactly like this:
```
"PB_QSLOT_BAD_ADDR,family"
```

Notice the location of the `,` - it's *within* the quotes.

Reply:
Once you add or modify the tests, the output of the test script will no longer match the `golden` output value. If you add a new clobber here in the array, you also have to modify `tests/golden/clobber-repair` to contain the new expected output so that it matches.